src/operator/custom/custom.cc

```diff
@@ -181,8 +181,18 @@ Operator* CustomOpProp::CreateOperatorEx(Context ctx, std::vector<TShape> *in_sh
 }
 
 MXNET_REGISTER_OP_PROPERTY(Custom, CustomOpProp)
-.describe("Custom operator implemented in frontend.")
-.add_argument("op_type", "string", "Type of custom operator. Must be registered first.");
+.describe(R"code(Apply a custom operator implemented in a frontend language (like Python).
+
+Custom operators should override required methods like `forward` and `backward`.
+The custom operator should be registered before it can be used.
+Please check the tutorial here: http://mxnet.io/how_to/new_op.html.
+
+)code")
+.add_argument("op_type", "string", "Name of the custom operator. "
+              "This is the name that is passed to `mx.operator.register` "
```

Comment:
"The custom operator should be registered before it can be used."

Is it more accurate to say it "should" be registered or that it *"must"* be registered?